ush/modify_yaml_syntax.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
```

Comment:
General question:  This script was used to convert files to replace `$( )` with `{{ }}`.  How long do we retain this script in the repo?  At some point all scripts use `{{ }}`.

Reply:
This is a good point. Do we even need this in the repository now?